README.md

```diff
@@ -1,5 +1,31 @@
 # Introduction 
-Project Coimbra
+Project Coimbra 
+
+# Setup for development
+1. Clone https://github.com/melanchall/drywetmidi/  , and put the repo in this structure 

```

Comment:
This isn't the right solution. You should use `git submodule init` instead and you won't need to worry about the structure then